02-account-kit-instantiate-usage/index.js

```diff
@@ -9,6 +9,9 @@ async function main() {
     // Retrieves data from the APIs (v1/chain/get_account)
     await account.loadData()
 
+    // Or do the above two steps in one step (instantiates object + calls load data)
+    const account: Account = Account.load('teamgreymass', Chains.EOS)
```

Comment:
This adds a new `.load` method on the `Account`, which will perform the two steps at the same time.